doc/apidoc/smac.epm.base_imputor.rst

```diff
@@ -0,0 +1,7 @@
+smac\.epm\.base\_imputor module
+===============================
+
+.. automodule:: smac.epm.base_imputor
```

Comment:
I've just pushed a commit to this branch adding some more documentation to the BaseImputor class. See 0d9c36e